change/@fluentui-react-f5fd080f-c867-41c4-be33-6b4e115a92ce.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31685&commitId=640e178d7dfc2525cf5b4f3ee81500987135b5eb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|6| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|5| Changed|

</details>


  <details><summary><b>Card Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.size.normal.chromium.png|0| Added|
|Card Converged.CardHeader.normal.chromium.png|0| Removed|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default - Dark Mode.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>